Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class Lesson3
+    {
+        public static void Demo()
+        {
+            string name = PromptString("Enter person´s name");
+            string surename = PromptString("Enter person´s surename");
+            int age = PromptInt("Enter person´s age");
+            float weight = PromptFloat("Enter person´s weight in kg");
+            float height = PromptFloat("Enter person´s height in m");
+            Console.WriteLine($"{Environment.NewLine}{name} {surename} is {age}, weight is {weight} kg and height is {height} cm.");
+            Console.WriteLine($"BMI is {CalculateBMI(weight, height):f2}");
```

Comment:
This block is repeated twice. Could it be a function? Other than that - looks good.